File: algorithms/Recursion/ToString/toStr.js
function toStr(number, base) {
    var convertString = "0123456789ABCDEF"
    if (number < base) {
        return convertString[number]
    } else {
        return toStr(Math.floor(number / base), base) + convertString[number % base]
    }
}

console.log(toStr(1435, 16))

File: algorithms/Recursion/ToString/toStr.go
package main

import (
	"fmt"
)

func toStr(number int, base int) string {
	convertString := "0123456789ABCDEF"
	if number < base {
		return string(convertString[number])
	} else {
		return toStr(number/base, base) + string(convertString[number%base])
	}
}

func main() {
	fmt.Println(toStr(1435, 16))
}
